axelrod/tests/strategies/test_mutual.py

```diff
@@ -20,12 +20,39 @@ class TestDesperate(TestPlayer):
     }
 
     def test_strategy(self):
-        self.first_play_test(C, seed=1)
-        self.first_play_test(D, seed=2)
-        self.responses_test([D], [C] * 4, [D] * 4)
-        self.responses_test([D], [D, D, C], [C, C, D])
-        self.responses_test([C], [D, D, D], [C, C, D])
+        self.first_play_test(C, seed = 1)
```

Comment:
For function arguments we don't include spaces around equal signs:
`self.first_play_test(C, seed=1)`

That's the guideline in PEP8. There are several more instances below.